tests/integration/custom-template/tests/__snapshots__/index.test.ts.snap

```diff
@@ -1,3 +1,3 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`custom template should allow to custom template by html.template option 1`] = `"<!doctype html><html><head><title>Hello World</title><script defer="defer" src="/static/js/builder-runtime.js"></script><script defer="defer" src="/static/js/lib-polyfill.js"></script><script defer="defer" src="/static/js/lib-react.js"></script><script defer="defer" src="/static/js/main.js"></script></head><body><div id="root"></div></body></html>"`;
+exports[`custom template should allow to custom template by html.template option 1`] = `"<!doctype html><html><head><title>Hello World</title><script defer="defer" src="/static/js/builder-runtime.js"></script><script defer="defer" src="/static/js/lib-polyfill.js"></script><script defer="defer" src="/static/js/lib-react.js"></script><script defer="defer" src="/static/js/911.js"></script><script defer="defer" src="/static/js/main.js"></script></head><body><div id="root"></div></body></html>"`;
```

Comment:
We can update the `modern.config.ts` of this case, and disable the chunk splitting:

```diff
export default defineConfig({
  html: {
    template: './src/index.html',
  },
  output: {
    disableFilenameHash: true,
    disableInlineRuntimeChunk: true,
  },
+  performance: {
+    chunkSplit: {
+      strategy: 'all-in-one',
+    },
+  },
  plugins: [appTools()],
});
```

After adding the above config, the HTML structure will be more stable, so we do not need to update the snapshot after making some changes.